dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckerTest.java

```diff
@@ -225,8 +225,8 @@ public void testNonExistentJarFileInput() throws ClassNotFoundException {
 
   @Test
   public void testCoordinateToJarPaths_validCoordinate() throws RepositoryException {
-    List<Path> paths = StaticLinkageChecker.coordinateToJarPaths("io.grpc:grpc-auth:1.15.1");
-    Truth.assertThat(paths).hasSize(11);
+    List<Path> paths = StaticLinkageChecker.coordinateToClasspath("io.grpc:grpc-auth:1.15.1");
```

Comment:
The test was updated because now the function builds complete dependencies including optional